tests/baselines/reference/classDeclarationBlockScoping2.js

```diff
@@ -0,0 +1,29 @@
+//// [classDeclarationBlockScoping2.ts]
+function f() {
+    class C {}
+    var c1 = C;
+    {
+        class C {}
+        var c2 = C;
+    }
+    return C === c1;
+}
+
+//// [classDeclarationBlockScoping2.js]
+function f() {
+    var C = (function () {
+        function C() {
+        }
+        return C;
+    })();
+    var c1 = C;
+    {
+        var C = (function () {

```

Comment:
So to be clear, you need to file a bug that we're not correctly renaming these classes.